pkg/policy/policy_controller.go

```diff
@@ -390,7 +390,7 @@ func (pc *PolicyController) enqueuePolicy(policy kyvernov1.PolicyInterface) {
 }
 
 // Run begins watching and syncing.
-func (pc *PolicyController) Run(workers int, reconcileCh <-chan bool, cleanupChangeRequest <-chan policyreport.ReconcileInfo, stopCh <-chan struct{}) {
+func (pc *PolicyController) Run(workers int, reconcileCh <-chan bool, cleanupChangeRequest <-chan string, stopCh <-chan struct{}) {
```

Comment:
Can we add a comment to describe what is passed in the string? It was not clear to me that this is the `namespace`.

Reply:
Sure

Reply:
fixed